SmartDeviceLinkTests/DevAPISpecs/SDLMenuManagerSpec.m

```diff
@@ -169,9 +176,28 @@ - (BOOL)sdl_shouldRPCsIncludeImages:(NSArray<SDLMenuCell *> *)cells;
         });
 
         context(@"duplicate titles", ^{
-            it(@"should fail with a duplicate title", ^{
-                testManager.menuCells = @[textOnlyCell, textOnlyCell];
-                expect(testManager.menuCells).to(beEmpty());
+            it(@"should not fail with a duplicate title", ^{
+                testManager.menuCells = @[textAndImageCell, textAndImageCell2];
+                expect(testManager.menuCells).toNot(beEmpty());
+                if ([[SDLGlobals sharedGlobals].rpcVersion isLessThanVersion:version]) {
+                    expect(testManager.menuCells.firstObject.uniqueTitle).to(equal("Test 2"));
+                    expect(testManager.menuCells.lastObject.uniqueTitle).to(equal("Test 2 (2)"));
+                } else {
```

Comment:
I don't see any place where the version is changed and this is re-tested. I think you need to make two separate tests, one with the contents of each part of the if, and then modify the pre-conditions to test each properly.